src/index.ts

```diff
@@ -21,7 +21,7 @@ export function hardhatConfigNetworks(): HardhatConfigNetworks {
   return CHAINS.reduce((networks, chain) => {
     networks[chain.alias] = {
       accounts: { mnemonic: '' },
-      chainId: chain.id,
+      chainId: parseInt(chain.id),
```

Comment:
```suggestion
      chainId: Number(chain.id),
```

Reply:
I figured `parseInt()` would throw when the string isn't an integer but it doesn't seem to be the case